schedule/sles4sap/publiccloud_hanasr.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: publiccloud_hanasr
+description: |
+  Run HANASR tests on public cloud instances using ha-sap-terraform-deployments tool:
```

Comment:
is it reference to ha-sap-terraform-deployments right here?

Reply:
Oh yes... thanks. Good catch. The schedule is quite old. Fixed.

Reply:
+1. Should be qe-sap-deployments, right?